stories/core/UserIconGroup.stories.tsx

```diff
@@ -200,15 +201,25 @@ export const WithTooltip: Story<UserIconGroupProps> = (args) => {
     'Justice Harrington',
   ];
 
-  const [tooltipVisible, setTooltipVisible] = React.useState<boolean>(false);
-
+  const [, setTooltipVisible] = React.useState<boolean>(false);
+  const userIconRef = React.useRef();
   const excessIconProps = {
+    id: 'someid',
+    ref: userIconRef,
     onMouseOver: () => setTooltipVisible(true),
     onMouseOut: () => setTooltipVisible(false),
   };
 
+  const arrayLength = 5;
+  const usersSubArray = userNames.slice(arrayLength);
+  const tooltipContent = usersSubArray.join(`\n`) as string;
+
   return (
-    <div className='iui-tooltip-container'>
+    <Tooltip
```

Comment:
You can place tooltip as sibling. #198 was merged

Reply:
Placed as sibling